clerk/backend/auth/auth.go

```diff
@@ -17,26 +20,22 @@ var secrets struct {
 //
 //encore:service
 type Service struct {
-	client clerk.Client
 }
 
 // initService is automatically called by Encore when the service starts up.
 func initService() (*Service, error) {
-	client, err := clerk.NewClient(secrets.ClientSecretKey)
-	if err != nil {
-		return nil, err
-	}
-	return &Service{client: client}, nil
+	clerk.SetKey(secrets.ClientSecretKey)
```

Comment:
Is the recommended approach to not generate a client? I kind of liked the approach of supplying the client to the service, without having to use global variables.